static/app/utils/theme/theme.chonk.tsx

```diff
@@ -792,8 +802,31 @@ export const DO_NOT_USE_darkChonkTheme: Theme = {
   stacktraceActiveBackground: darkTheme.stacktraceActiveBackground,
   stacktraceActiveText: darkTheme.stacktraceActiveText,
 
+  colors: {
+    light: lightColors,
+    dark: darkColors,
+  },
+
+  space,
+
   sidebar: {
     // @TODO: these colors need to be ported
     ...darkTheme.sidebar,
   },
 };
+
+declare module '@emotion/react' {
+  // @TODO(jonasbadalic): interface extending a type might be prone to some issues.
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  export interface DO_NOT_USE_ChonkTheme extends ChonkTheme {
+    isChonk: true;
+  }
```

Comment:
is it still necessary that we expose it like this? I’m not sure where an interface called `DO_NOT_USE_ChonkTheme` would be used from `@emotion/react` 🤔